app/views/birds/show.html.erb

```diff
@@ -8,5 +8,40 @@
   <%= @bird.user.username %>
 </p>
 
+<table>
+  <thead>
+    <tr>
+      <th>Comment</th>
+      <th>User</th>
+      <th>Date</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @bird.comments.reverse.each do |comment| %>
+      <tr class="<%= cycle('list_line_odd', 'list_line_even')%>">
+        <td><%= comment.body %></td>
+        <td><%= comment.user.username %></td>
+        <td><%= comment.created_at.strftime("%d/%m/%Y\n%k:%M") %></td>
+        <% if current_user == comment.user || current_user == @bird.user %>
+          <td><%= link_to 'Destroy', '/birds/:bird_id/comments/:id(.:format)' , method: :delete %></td>
```

Comment:
let's use only the rails helpers for the paths. instead of adding the url as a string, use the route method that rails generates for you. `localhost:3000/rails/info/routes`